plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/client/BackpressureRestHighLevelClient.java

```diff
@@ -84,28 +84,46 @@ public void close()
         delegate.close();
     }
 
-    public SearchResponse search(SearchRequest searchRequest, Header... headers)
```

Comment:
Is it possible to split them into two methods - First we could remove the unused `Header...` and then we could remove the deprecated APIs. This makes it a bit difficult on identifying which is deprecated API. WDYT ?

Reply:
You meant to commits, right? 
Done